CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 - `loadbalancingexporter`: Allow non-exist hostname on startup (#7935)
 - `datadogexporter`: Use exact sum, count and average on Datadog distributions (#7830)
 - `storage/filestorage`: add optional compaction to filestorage (#7768)
+- `hostreceiver/processscraper`: Migrate the scraper to the mdatagen metrics builder (#7287)
```

Comment:
v0.45.0 just got released. Please rebase and move this line to Unreleased

Reply:
Done